### PR TITLE
Phil e/5

### PR DESCRIPTION
**Pull request by phiddle**

### CODE REVIEW

**Review by jthomasw88 — DISMISSED** [review]
looks good to me!!

**Review by Edwine-R — DISMISSED** [review]

**Review by nisharai26 — APPROVED** [review]
done!!

**Review by JakeyD11 — APPROVED** [review]

**Review by jadavey91 — APPROVED** [review]
Nice

File: phiddle/index.html [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+        </th>
+        <th>
+            <h1>Film</h>
+        </th>
+        <th>
+            <h1>Actor</h1>
+        </th>
+    </tr>
+    <br>
+    <br>
```
Comment (nmunro) [inline phiddle/index.html:18]: It's not generally a good idea to put breaks inside tables

File: phiddle/index.html [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+        </td>
+        <td>
+            <h3>The Wolf of Wall Street</h3>
+        </td>
+        <td>
+            <h3>Leoarndo DiCaprio</h3>
+        </td>
+    </tr>
+    <br>
+    <br>
```
Comment (nmunro) [inline phiddle/index.html:31]: Same here

File: phiddle/index.html [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+            <h3>Action</h3>
+        </td>
+        <td>
+            <h3>Rogue One</h3>
+        </td>
+        <td>
+            <h3>Felicity Jones</h3>
+        </td>
+    </tr>
+    <br>
```
Comment (nmunro) [inline phiddle/index.html:43]: And here

File: phiddle/prompt.js [hunk 4]
```diff
@@ -0,0 +1,3 @@
+let name = window.prompt("whats your name?");
```
Comment (nmunro) [inline phiddle/prompt.js:1]: The window object isn't required here, prompt is enough :)

File: phiddle/prompt.js [hunk 5]
```diff
@@ -0,0 +1,3 @@
...
+
+window.alert(`Hi  ${name}`);
```
Comment (nmunro) [inline phiddle/prompt.js:3]: Same here, the alert function on its own is enough

**Review by nmunro — APPROVED** [review]

**Pull request merged**